modules/features/manatime_feature_projects/manatime_feature_projects.module

```diff
@@ -43,6 +44,48 @@ function manatime_feature_projects_action_info() {
 }
 
 /**
+ * Implements hook_field_widget_form_alter().
+ */
+function manatime_feature_projects_field_widget_form_alter(&$element, &$form_state, $context) {
+  $query = drupal_get_query_parameters();
+  // Query string param.
+  if (!empty($query['field_project'])) {
+    unset($element['actions']['ief_add']);
+    unset($element['actions']['ief_add_existing']);
+    // Hide action buttons.
+    if (isset($element['#field_name']) && 'field_project' == $element['#field_name']) {
+      if (!isset($element['entities'][0])) {
+        $delta = 0;
+        $project = array($query['field_project']);
+        foreach ($form_state['inline_entity_form'] as $key_state => $form_state_item) {
+          if ('projects' == $form_state_item['settings']['bundles'][0]) {
+            break;
+          }
+        }
+        $entity_list = entity_load('node', $project);
```

Comment:
change `entity_load` to `node_load` and change variable name because we're sure there's only one entity (and it's a project node)